lxd/network/driver_ovn.go

```diff
@@ -5887,3 +5907,76 @@ func (n *ovn) checkAddressNotInUse(netip *net.IPNet) (bool, error) {
 
 	return true, nil
 }
+
+// checkAddressNotInOVNRange checks that a given IP address does not overlap
+// with OVN ranges set on the uplink network. However, it allows an IP address
+// that is the same as volatile network address of the current network.
+// Returns an error if the check could not be performed or the IP address
+// overlaps with OVN ranges.
+func (n *ovn) checkAddressNotInOVNRange(addr net.IP) error {
+	if addr == nil {
+		return fmt.Errorf("Invalid listen address")
+	}
+
+	if n.config["network"] == "" {
+		return fmt.Errorf(`OVN network %q is missing "network" config option`, n.name)
+	}
+
+	addrIsIP4 := addr.To4() != nil
```

Comment:
just a nit but i would have moved that down to where its tested on line 5930 to keep it together.